docs/docs/middlewares/path.md

```diff
@@ -0,0 +1,41 @@
+# Path
+This component returns the URL for the given route. 
+
+## Parameters
+```path(page, queries, locale)```
+
+parameter        | type           | required | description
```

Comment:
I think the types and/or the descriptions are wrong

Reply:
@mbohal This is not very useful, hope you’ll agree. I tried to describe the mistakes below, could you please have a look at it to check that I didn’t miss anything?